objectstripe.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto2";
```

Comment:
I didn't want to hand-pack stuff, and (although probably big-endian systems will never run this) in principle this is safer. Perhaps arrow would be a better choice though. (or even ROOT I suppose)

Reply:
yeah this is google protobuf